projects/jobs/jobs.groovy

```diff
@@ -291,6 +292,7 @@ xmlFiles.each {
 ''')
   environmentVariables {
       env('PLUGGABLE_SCM_PROVIDER_PATH','${WORKSPACE}/job_dsl_additional_classpath/')
+      env('Custom_Cartridge_Library_PATH','${JENKINS_HOME}/userContent/job_dsl_additional_classpath/adop/cartridge/properties/')
```

Comment:
Please can we use the standard naming convention e.g. all uppercase for env variables.

Reply:
We don't need to set Custom_Cartridge_Library_PATH since it's not actually used so we can remove it. The classpath is just inherited from the pluggable SCM classpath. We should just retitle PLUGGABLE_SCM_PROVIDER_PATH to JOB_DSL_ADDITIONAL_CLASSPATH and ensure it replaces PLUGGABLE_SCM_PROVIDER_PATH everywhere.